docs/changelog.rst

```diff
@@ -123,6 +129,14 @@ Cleanups
 - Catch errors when loading :confval:`logging/config_file`.
   (Fixes: :issue:`1320`)
 
+Audio
+-----
+
+- **Breaking:** The audio scanner now returns ISO-8601 formatted strings
```

Comment:
I think I looked into Mopidy-Local-* at some point looking for issues with this without finding anything, but it would be great if @tkem checked himself.